src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -252,9 +264,10 @@ Copyright (c) .NET Foundation. All rights reserved.
       <Output TaskParameter="RuntimeTargets" ItemName="RuntimeTargetsCopyLocalItems" />
       <Output TaskParameter="CompileTimeAssemblies" ItemName="ResolvedCompileFileDefinitions" />
       <Output TaskParameter="TransitiveProjectReferences" ItemName="_TransitiveProjectReferences" />
+      <Output TaskParameter="PackageFolders" ItemName="AssetFilePackageFolder" />
```

Comment:
We have AssetsFile (plural) not AssetFile (singular) elsewhere and we should be consistent.